src/Symfony/Component/Config/ConfigCache.php

```diff
@@ -44,6 +44,7 @@ public function __construct($file, $debug)
      * Gets the cache file path.
      *
      * @return string The cache file path
+     * @deprecated since 2.7, to be removed in 3.0. Use getFilePath() instead.
      */
     public function __toString()
     {
```

Comment:
You need to add a deprecation notice (look for `trigger_error()` in Symfony 2.7 for some examples).